### DISCUSSION TITLE
Execute shell command from AntiMicroX?

### DISCUSSION THREAD

**Discussion by EramarkMedia** [AntiMicroX/antimicrox#590] [Q&A] [answered]
Good day.

I am using a Xbox 360 controller on Ubuntu, in the living room, and a couple other places.
As it is one of the most compact controller solutions, in my taste.

I tried getting the chatpad / keyboard to work at some point, without success.
But then instead thought, i will just go with the onscreen keyboard instead.

And well, this works fine in 98% of the cases.
However, some applications like Firefox etc, don't seem to enable the onscreen keyboard, when their text fields are selected for input.

Hence, i would like to know, if i could map a controller action, to execute a shell command, in order to force the keyboard to draw / hide.

Thank you in advance, and have a good weekend.

**Comment by pktiuk** [comment, marked as answer]
Yes, you can map button press to shell script.

**Reply by EramarkMedia** [reply]
@pktiuk May i ask how and where to do this?
Is it simply in the commands tab?

If so, would i have to do something like ```$command```or similar?

Thank you in advance.

**Reply by pktiuk** [reply]
Use `Advanced` assignments.

**Reply by EramarkMedia** [reply]
Had a look yesteday, and figured it out after you'r reply.
I was being confused, by the prompt, that accessing advanced settings, needed to have a key bound to it, before being able to access the menu.

Thank you.
I can now show / hide the onscreen keyboard, from the Xbox controller.
